src/main/java/domain/calculator/exception/InvalidInputException.java

```diff
@@ -0,0 +1,15 @@
+package domain.calculator.exception;
+
+public class InvalidInputException extends RuntimeException {
+
+    private String message;
+
+    public InvalidInputException(String message) {
+        this.message = message;
+    }
+
+    @Override
+    public String getMessage() {
+        return message;
+    }
```

Comment:
아래와 같이 사용해도 의도한 바와 동일하게 동작할것 같네요 🙂

```suggestion

    public InvalidInputException(String message) {
        super(message);
    }
```